dashboard/app/config.go

```diff
@@ -121,6 +121,14 @@ type Config struct {
 	DisplayDiscussions bool
 	// Cache what we display on the web dashboard.
 	CacheUIPages bool
+	// Enables coverage aggregation
+	Coverage *CoverageConfig
+}
+
+type CoverageConfig struct {
+	ShowPage            bool
```

Comment:
Is there any scenario where we want CoverageConfig object to be present, but ShowPage set to false? If not, I would remove this field.

Reply:
Mmm... I'd like to do it for internal kernels.
The price of "ShowPage" flag removal will be the empty pages which looks acceptable.

Reply:
Done.